html/index.html

```diff
@@ -1,16 +1,17 @@
 ﻿<!DOCTYPE HTML>
 <html>
 
-  <head>
-    <meta charset="utf-8"/>
+<head>
+    <meta charset="utf-8" />
+    <meta name="description" content="Improved dump1090-fa web interface">
```

Comment:
I don't think this is a good description, just my opinion. How about: `Tar1090 - Live flight tracking map brought to you by ADS-B Exchange`? Or something a little more SEO-ey? :)

Reply:
I agree it sounds much better but just copied it from this repo readme. I think it is up to wiedehopf to change it.